packages/mdc-button/_mixins.scss

```diff
@@ -64,12 +64,20 @@
   // Note: line height is adjusted for stroke button because borders are not
   // considered as space available to text on the Web
   line-height: $mdc-button-height - $stroke-width * 2;
+  $padding-value: max($mdc-button-horizontal-padding - $stroke-width, 0);
```

Comment:
Might want a comment above this too, e.g. "Adjust padding to maintain consistent width with non-stroked buttons"

:+1: to centralizing this though; this actually makes the demo page line up nicer, in addition to allowing for stroke width changes across states for a single button without affecting its width.